### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by helpmepleasebro21** [lichess-bot-devs/lichess-bot#886] [General]
The Bot doesn't work but the old lichess-bot-master works perfectly on windows 10 please help

**Comment by helpmepleasebro21** [comment]
![image](https://github.com/lichess-bot-devs/lichess-bot/assets/155887882/f2f29547-67d3-41e5-865e-58bd63734926)
what to do?

**Reply by MarkZH** [reply]
From your screenshot, it appears you don't have a config.yml file, nor do you specify a different file with the `-c` option. Do you have a configuration file? If not, look in the wiki for help starting with the last instruction (Copy config.yml.default to config.yml.) here: [Installation instructions](https://github.com/lichess-bot-devs/lichess-bot/wiki/How-to-Install#windows)

Let us know if you need more help.
